Assets/_Scripts/Units/Player/Customization/PlayerCustomization.cs

```diff
@@ -40,7 +42,7 @@ public override void Spawned()
             base.Spawned();
             
             settings = SettingsSystem.CustomizationSettings;
-            if (Object.HasInputAuthority) Randomize(true);
+            if (Object.HasInputAuthority) Randomize(randomizeGameplayElementsAtStart);
```

Comment:
Why would we want this to be true? On a mis que tous le monde commence avec la classe de base donc les vetements ne devrait pas être changé de ceux de base.

Reply:
Je savais pas trop ce qu'on voulait faire avec l'AI. Maintenant que je sais que tout le monde commence avec la classe de base, je vais juste revert et on va pas randomizer les vêtements